cypress/e2e/systemIntake.cy.js

```diff
@@ -208,10 +216,18 @@ describe('The System Intake Form', () => {
       .type('Mock Current Annual Spend')
       .should('have.value', 'Mock Current Annual Spend');
 
+    cy.get('#IntakeForm-CurrentAnnualSpendingITPortion')
+      .type('Mock Current Annual Spend IT Portion')
+      .should('have.value', 'Mock Current Annual Spend');
```

Comment:
```suggestion
      .should('have.value', 'Mock Current Annual Spend IT Portion');
```

Looks like this was maybe just a copy-paste error, but it does cause E2E to fail!